eval-config.nix

```diff
@@ -1,10 +1,75 @@
+let
+  nixDarwinVersion = builtins.fromJSON (builtins.readFile ./version.json);
+
+  checkRelease = lib:
+    # Avoid breaking configurations when the unstable Nixpkgs version
```

Comment:
Is this something we need to avoid? I assume we would just want to cut a release whenever NixOS makes a release, although I'm not sure how much effort that will be, probably best to make a guide when you do it for the first time with 24.11

Reply:
The problem with unconditionally using `==` is that as soon as 24.11 gets branched off and `master` is updated to 25.05, we’d start throwing an error on our `master` branch. We would have to bump the release in perfect lockstep with Nixpkgs (actually, in perfect lockstep with both `nixpkgs-unstable` and `nixos-unstable`, which is not necessarily possible – one can become 25.05 before the other). That would be more possible with automation, but I’ve punted on the automation for now to get this out the door.

This is a perennial problem with Home Manager, which only makes it a warning – every branch‐off, it starts printing spurious warnings. Just doing a conservative check for now seemed like the best thing; we can bump to 25.11 after both `nixpkgs-unstable` and `nixos-unstable` get it.